### DISCUSSION TITLE
Add support for RAW+JPEG pairs

### DISCUSSION THREAD

**Discussion by randolph-** [ente-io/ente#2276] [Enhancements]
Hi, Ente folks! Many cameras allow photographers to simultaneously shoot in RAW and JPEG formats. In other photo management applications, [these two files are often displayed as a single item](https://github.com/ente-io/ente/assets/6894475/efdb0eb1-4e04-461d-9a70-ff33a15a83a7), while Ente currently [handles the two files as separate photos](https://github.com/ente-io/ente/assets/6894475/56af7f94-80fc-417c-b49b-1bad14a9effc). It isn't a showstopper feature by any means, but it would be great if Ente could combine these two photos—similar to how Ente currently combines the HEIC and MOV components of a Live Photo—into a singular library item. Cheers!

**Comment by LNA-DEV** [comment]
This even is a thing if you have enabled RAW on a phone. In the current state of Ente every image I take (with my phone or otherwise) is duplicated in the gallery...

Would be really cool to have such sort of grouping!

**Reply by nikos-95** [reply]
Greatly support this suggestion. 
Also, as an alternative to merging the files, it would be viable to filter the shown+synced file types, e.g. to only display jpg (in the app).
